doc/whats_new/v0.21.rst

```diff
@@ -55,6 +55,13 @@ Support for Python 3.4 and below has been officially dropped.
   of calculating it every time on the fly.
   :issue:`12116` by :user:`Ekaterina Krivich <kiote>` and `Joel Nothman`_.
 
+- |Feature| |Efficiency| :class:`preprocessing.PolynomialFeatures` now supports
```

Comment:
I don't think we should say feature here. It is not a new capability in the software